src/main/java/org/sopt/util/PostWriteLimiter.java

```diff
@@ -0,0 +1,27 @@
+package org.sopt.util;
+
+import java.time.LocalDateTime;
+
+public class PostWriteLimiter {
+
+    private static LocalDateTime lastPostedTime;
+    private static final int timeLimit = 3;
```

Comment:
상수는 일반적으로 대문자 와  _ 기호를 조합해서 명명합니다!
```suggestion
    private static final int TIME_LIMIT = 3;
```